### ISSUE TITLE
Did you train with wing loss? I see that you wrote this loss function, but it was not used, why?

### ISSUE THREAD

**Issue by DuckJ** [ElvishElvis/68-Retinaface-Pytorch-version#5]

**Comment by ElvishElvis** [comment]
I trained with many different losses throughout the process, including wing loss, L1 loss and so on.
Wing loss is used to finetune the model at the end.

**Comment by DuckJ** [comment]
finetune? Don't you train from scrath？
I may have misunderstood
Do you think wingloss could replace smoothl1

**Comment by ElvishElvis** [comment]
I started from scrath with losses like smooth L1/ L1, until the loss is very small. ( like, less than 1, thus 0.5*(X^2) portion of smooth L1 would not work since the square of something less than 1 would only make it smaller. Thus tiny losses will be ignored. Thus it's when we should switch. 

The purpose of Wing loss/ Adpative Wing loss is to fine-tune the landmarks loss when it's very small. Also, we need to adjust the weight of face, bbox and landmark headers accordingly throughout the training.

**Comment by DuckJ** [comment]
Thank you for your reply
